docs/html/user_guide.rst

```diff
@@ -493,10 +493,10 @@ to PyPI.
 
 First, download the archives that fulfill your requirements::
 
-$ pip install --download DIR -r requirements.txt
+$ pip download DIR -r requirements.txt
```

Comment:
It's either `pip download -r requirements.txt` or `pip download --destination-directory DIR -r requirements`
(cf https://github.com/pypa/pip/blob/35701ea3cd41eb62c2e9e13bf5ba16ce960bd8dc/src/pip/_internal/commands/download.py#L62-L68)

Reply:
Thanks for the review. The original example meant downloading to a DIR so I will change it to `pip download --destination-directory DIR -r requirements.txt`

pip 9.0.1 behaviour

```
(scratch-env) ➜  foo pwd
/tmp/foo
(scratch-env) ➜  foo cat requirements.txt
yapf==0.14.0
(scratch-env) ➜  foo mkdir bar
(scratch-env) ➜  foo pip install --download bar -r requirements.txt
DEPRECATION: pip install --download has been deprecated and will be removed in the future. Pip now has a download command that should be used instead.
Collecting yapf==0.14.0 (from -r requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/fc/39/9e58e20b462acde483948955c564dbda3e912eff572da15f64496280bfd2/yapf-0.14.0-py2.py3-none-any.whl
  Saved ./bar/yapf-0.14.0-py2.py3-none-any.whl
Successfully downloaded yapf
```

pip 18.1 behavior

```
(scratch-env) ➜  foo pip install -U pip
Cache entry deserialization failed, entry ignored
Collecting pip
  Using cached https://files.pythonhosted.org/packages/c2/d7/90f34cb0d83a6c5631cf71dfe64cc1054598c843a92b400e55675cc2ac37/pip-18.1-py2.py3-none-any.whl
Installing collected packages: pip
  Found existing installation: pip 9.0.1
    Uninstalling pip-9.0.1:
      Successfully uninstalled pip-9.0.1
Successfully installed pip-18.1
(scratch-env) ➜  foo mkdir baz
(scratch-env) ➜  foo pip download --destination-directory baz -r requirements.txt
Collecting yapf==0.14.0 (from -r requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/fc/39/9e58e20b462acde483948955c564dbda3e912eff572da15f64496280bfd2/yapf-0.14.0-py2.py3-none-any.whl
  Saved ./baz/yapf-0.14.0-py2.py3-none-any.whl
Successfully downloaded yapf
```